issues.js

```diff
@@ -1,6 +1,12 @@
 const config = require('./config'),
   octokit = require('@octokit/rest')({ headers: config.headers });
 
+
+const removeExcludedLabels = (issues) => {
+  return issues.filter(issue => {
```

Comment:
If the RHS of arrow function just contains a single statement, it's probably better to write that without `{}` (short hand way)
For example this line could become:

```
issues.filter(issue => !issue.labels.some(label => config.excludeLabels.includes(label.name.toLocaleLowerCase())))
```